pyclam/manifold.py

```diff
@@ -1198,7 +1205,8 @@ def distance(self, x1: Union[List[int], Data], x2: Union[List[int], Data]) -> np
 
     def build(self, *criteria) -> 'Manifold':
         """ Rebuilds the Cluster-tree and the Graph-stack. """
-        from pyclam.criterion import ClusterCriterion, SelectionCriterion, GraphCriterion
+        from pyclam.criterion import ClusterCriterion, SelectionCriterion
```

Comment:
weird to lose graph criterion in build

Reply:
Yeah the names probably need some improvement. GraphCriterion are for fine-tuning of graphs. SelectionCriterion are for building the actual graphs. GraphCriterion will make a comeback one I fix add and remove for Graphs